### PR TITLE
Create natural stork

### PR DESCRIPTION
**Pull request by aeslaughter**
Since very few people will ever create a module, the stork script just assume we are creating an app unless told otherwise. 
(refs #8266)

### CODE REVIEW

File: scripts/stork.sh [hunk 1]
```diff
@@ -1,28 +1,33 @@
 #!/bin/bash
 
 function printusage {
-    echo "Usage:    stork.sh <type> <AppName>"
+    echo "Usage:    stork.sh <AppName>"
```
Comment (andrsd) [inline scripts/stork.sh]: Do you want to change `<AppName>` to  `<Name>`?

Comment (aeslaughter) [inline scripts/stork.sh]: Do you want me to change <AppName> to <Name>?

Comment (aeslaughter) [inline scripts/stork.sh]: @brianmoose If this goes in you will probably need to update the stork testing.

Comment (andrsd) [inline scripts/stork.sh]: If you think it is a good idea, then yes.

**Comment by friedmud** [comment]
This idea gets a :+1: from me

**Comment by brianmoose** [comment]
I have a fix for the stork test once this is merged.

**Review by permcody — APPROVED** [review]
Looks good! Thanks

**Comment by permcody** [comment]
Failed stork test is expected, a change to stork will go in after this merge.

**Pull request merged**